deploy/pivot_role/pivotRole.yaml

```diff
@@ -455,7 +463,7 @@ Resources:
               - 'cloudformation:DescribeStackResource'
             Resource:
               - !Sub 'arn:aws:cloudformation:*:${AWS::AccountId}:stack/${EnvironmentResourcePrefix}*/*'
-              - !Sub 'arn:aws:cloudformation:*:${AWS::AccountId}:stack/CDKToolkit/*'
+              - !Sub 'arn:aws:cloudformation:*:${AWS::AccountId}:stack/CDKToolkitDataAll/*'
```

Comment:
Can you also include
- !Sub 'arn:aws:cloudformation:*:${AWS::AccountId}:stack/CDKToolkit/*' ?

Reply:
Yes sure, this line was not intended to be changed upstream as we use a separate workspace naming only in our fork.